libs/algorithms/tests/unit/algorithms/generate.cpp

```diff
@@ -187,7 +245,7 @@ void generate_exception_test()
 
 //////////////////////////////////////////////////////////////////////////////
 template <typename ExPolicy, typename IteratorTag>
-void test_generate_bad_alloc(ExPolicy policy, IteratorTag)
+void test_generate_bad_alloc(ExPolicy&& policy, IteratorTag)
```

Comment:
@hkaiser  Why no tests for bad_alloc without exec-policy?

Reply:
I was too lazy to add those ;-) (I have not added those for any of the ported algorithms, sorry.